emekafredy/maintenance-tracker
# Ft login endpoint 157657624

Description of Task to be completed?

- Setup database connection
- Write tests for endpoints
- Create POST route to api/v1/auth/signup
- Create POST route to api/v1/auth/login
- Validate user inputs
- Persist data
- Test using Postman

How should this be manually tested?

- Clone the repo
- Startup the command line
- run "npm install"
- run  "npm run mig"
- run  "npm start"
- Test endpoint using Postman

Any background context you want to provide?
- N/A

What are the relevant pivotal tracker stories?
[157730574](https://www.pivotaltracker.com/story/show/157730574)
[157657512](https://www.pivotaltracker.com/story/show/157657512)
[157657624](https://www.pivotaltracker.com/story/show/157657624)

What are the relevant Github Issues?
N/A

Questions:
N/A
